srcpkgs/sqlitestudio/files/sqlitestudio.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=SQLiteStudio
+Comment=Database manager to create, edit, and browse SQLite databases
+Icon=/usr/share/icons/hicolor/scalable/apps/sqlitestudio.svg
+Exec=sqlitestudio %f
+Terminal=false
+Categories=Database;Development;Utility;
+MimeType=application/vnd.sqlite3;application/x-sqlite3;
```

Comment:
Could you open a PR upstream to include this .desktop file in their repository? It would be nice if every distribution didn't have to ship their own.

Reply:
I looked at doing that, but they have a desktop entry contained within an [XML file](https://github.com/pawelsalawa/sqlitestudio/blob/master/SQLiteStudio-installer.xml) which they're using for installation, so it looked unlikely to be accepted. You think I should still give it a try there?